plugin-scalebar/src/test/java/com/mapbox/maps/plugin/scalebar/ScaleBarImplTest.kt

```diff
@@ -98,25 +99,29 @@ class ScaleBarImplTest {
 
   @Test
   fun getDistanceText() {
-    assertEquals("0", scaleBarView.getDistanceText(0))
-    assertEquals("100 m", scaleBarView.getDistanceText(100))
-    assertEquals("999 m", scaleBarView.getDistanceText(999))
-    assertEquals("1 km", scaleBarView.getDistanceText(1000))
-    assertEquals("1 km", scaleBarView.getDistanceText(1001))
-    assertEquals("4.4 km", scaleBarView.getDistanceText(4444))
-    assertEquals("5.6 km", scaleBarView.getDistanceText(5555))
-    assertEquals("10 km", scaleBarView.getDistanceText(10000))
+    assertEquals("0", scaleBarView.getDistanceText(0F))
```

Comment:
nice to add parametrized tests. same for `verify below and above scale bar segments`

Reply:
Please see 69fb987 and baca659 and let me know if that's what you were suggesting.